### DISCUSSION TITLE
Manifest clarificaton

### DISCUSSION THREAD

**Discussion by sreejesh-radhakrishnan-db** [strimzi/strimzi-kafka-operator#7894] [General]
HI I can see in here  https://github.com/strimzi/strimzi-kafka-operator/tree/main/install      

topicOperator and user-operaotr folder (with manifest names 01*.yaml to 05*.yaml )

But I cannot find these manifest atleast those names  01*.yaml , 02*.yaml ,03*.yaml  on  https://github.com/strimzi/strimzi-kafka-operator/tree/main/helm-charts/helm3/strimzi-kafka-operator   

My Q - are folder and manifest for user and topic needed only if I am trying to install these in isolation and now use standard helm chart deployment?

**Comment by scholzj** [comment]
There is no Helm Chart for standalone Topic and User operators. The Helm Chart supports only the installation of the Cluster Operator.

**Reply by sreejesh-radhakrishnan-db** [reply]
thank you, sorry if I may ask what is the usecase to install topic and user operator separate ?    cant I get the benifit of topic and user operator is I just install cluster Operator?

**Reply by scholzj** [reply]
If you are using Strimzi managed Kafka cluster, you should always deploy the Topic and User operators through the Cluster Operator and the Kafka custom resource. But some users for example want to use the Topic or User operators against their own Kafka clusters not managed by Strimzi. In that case, they do not have the `Kafka` custom resource - but they can use the standalone installations.
